src/components/WebexMeetingControlBar/WebexMeetingControlBar.jsx

```diff
@@ -121,15 +128,15 @@ export default function WebexMeetingControlBar({
   ]);
   /* eslint-enable react-hooks/exhaustive-deps */
 
-  const renderControls = (names) => names.map(
+  const renderControls = (names, controlIndexes) => names.map(
     (key, i) => (
```

Comment:
the `i` here is an index in the sub-array of controls produced by `slice`, however I see that we're using `controlTexts[i]` and `controlsTabIndexes[i]` for computing `showText` and `autoFocus`, which does not seems right as controlTexts and controlsTabIndexes are containing the full list of controls.

My suggestion would be to change this function (renderControls) to receive the start and end indexes of the controls to render (not a sub-array), and use these indexes to extract information from controlNames, controlTexts and controlsTabIndexes.

```js
renderControls(start, end) => range(start, end).map((i) => (
  key = {names[i]}
  showText = {controlTexts[i]}
  tabIndex = {controlsTabIndexes[i]}
)
```
where `range` is a function that produces a list of numbers from `start` to `end-1`, eg:
```js
range = (start, end) => Array(end-start).fill().map((_, i) => start+i)

Reply:
And I think it's ok, for simplicity, to have a gap in the tab indexes list when some are collapsed, they don't really need to be consecutive: 1, 2, 5, 6, 7. The collapse button would have the tab index 3 in this case.